source/javascripts/services/_appService.js.erb

```diff
@@ -283,6 +287,27 @@
 					appService.savedDefaultMachineType = angular.copy(appService.defaultMachineType);
 				}
 
+				if (
+					requestService.mode === "website" &&
+					appService.defaultLicensePoolId
```

Comment:
If the user unsets the default license pool ID, this `if` will not run, thus [unsetting on save](https://github.com/bitrise-io/bitrise-workflow-editor/pull/802/files#diff-34c45354114ff878b352674d21c5a0c6914d86abbf5b64974b99186219b31550R305) will not happen.

My first guess is that this should be `requestService.mode === "website" && (appService.defaultLicensePoolId || appService.savedDefaultLicensePoolId)` so it's executed either if the field IS defined, or if it WAS defined.

Reply:
removed all default license pool related changes